jupyter_server/base/handlers.py

```diff
@@ -603,7 +614,7 @@ async def prepare(self):
                 DeprecationWarning
                 # stacklevel not useful here
             )
-            user = self.get_current_user()
+            user = User(self.get_current_user())
```

Comment:
`User` is only imported when typechecking is on, so this actually causes an invalid import error.